More fixes for Makie (#303)
JuliaDataScience/JuliaDataScience: contents/data_vis_makie_create_figure.md

The basic container object in Makie is `Figure`, a canvas where we can add objects like `Axis`, `Colorbar`, `Legend`, etc.

For `Figure`, we have have attributes like `backgroundcolor`, `resolution`, `font` and `fontsize` as well as the `figure_padding` which changes the amount of space around the figure content, see the colored area in @fig:fig_axis.
It can take one number for all sides, or a tuple of four numbers for left, right, bottom and top. Let's dive into these components.

### Figure

```jl
@sco JDS.figure_canvas()
```

### Figure + Axis

`Axis` has a lot more, some of them are  `backgroundcolor`, `xgridcolor` and `title`.
For a full list just type `help(Axis)`.

```jl
@sco JDS.figure_axis()
```

You can hide decorations and axis' spines with:

> - `hidedecorations!(ax; kwargs...)`
> - `hidexdecorations!(ax; kwargs...)`
> - `hideydecorations!(ax; kwargs...)`
> - `hidespines!(ax; kwargs...)`

Remember, we can always ask for help to see what kind of arguments we can use, e.g.,

```jl
s = """
    help(hidespines!)
    """
sco(s)
```

Alternatively, for decorations

```jl
s = """
    help(hidedecorations!)
    """
sco(s)
```

> For elements that **you don't want to hide**, just pass them with `false`, i.e. `hideydecorations!(ax; ticks=false, grid=false)`.

Reviewer: ```suggestion
For elements that **you don't want to hide**, pass `false`, e.g., `hideydecorations!(ax; ticks=false, grid=false)`.
```
i.e. means "in other words", whereas "e.g." means "for example". "for example" seems more appropriate here
